.github/workflows/build_all.yml

```diff
@@ -134,37 +144,39 @@ jobs:
           fab release package:destroy_vm=True
           mkdir magma-packages
           vagrant ssh -c "cp -r magma-packages /vagrant"
+      - name: Setup JFROG CLI
+        uses: jfrog/setup-jfrog-cli@d0a59b1cdaeeb16e65b5039fc92b8507337f1559 # pin@v3
       - name: Publish debian packages
         id: publish_packages
         if: github.event_name == 'push'
+        env:
+          DEBIAN_META_INFO: deb.distribution=focal-ci;deb.component=main;deb.architecture=amd64
         run: |
-          cd lte/gateway/magma-packages
+          # Default firebase output before anything can fail
           ARTIFACTS='{"packages":[],"valid":false}'
-          PUBLISH_ERROR="false"
-          for i in `ls -a1 *.deb`
-          do
-            echo "Pushing package $i to JFROG artifiactory: https://artifactory.magmacore.org/artifactory/debian-test/pool"
-            HTTP_RESPONSE=$(curl --silent --write-out "HTTPSTATUS:%{http_code}" -uci-bot:${{ secrets.JFROG_CIBOT_APIKEYS }} -XPUT "https://artifactory.magmacore.org/artifactory/debian-test/pool/focal-1.8.0/$i;deb.distribution=focal-1.8.0;deb.component=main;deb.architecture=amd64" -T $i)
-            echo "$HTTP_RESPONSE"
-            # extract the body and download uri
-            HTTP_BODY=$(echo $HTTP_RESPONSE | sed -e 's/HTTPSTATUS\:.*//g')
-            URI=$(echo $HTTP_BODY | jq -r '.uri')
-            if [[ "$URI" != "null" ]]; then
-              ARTIFACTS=$(echo $ARTIFACTS | jq --arg uri $URI '.packages += [$uri]')
-            else
-              PUBLISH_ERROR="true"
-            fi
-            # extract and check status
-            HTTP_STATUS=$(echo $HTTP_RESPONSE | tr -d '\n' | sed -e 's/.*HTTPSTATUS://')
-            if [[ "$HTTP_STATUS" != "2"* ]]; then
-              PUBLISH_ERROR="true"
-            fi
-          done
-          # set output
-          if [[ "$PUBLISH_ERROR" != "true" ]]; then
-            ARTIFACTS=$(echo $ARTIFACTS | jq '. += {"valid":true}')
-          fi
-          echo "::set-output name=artifacts::$(echo $ARTIFACTS)"
+          echo "artifacts=$(echo $ARTIFACTS)" >> $GITHUB_OUTPUT
```

Comment:
Why is the extra echo needed?

Reply:
Same as above

Reply:
Just out of curiosity, do you know why it is needed?